.travis.yml

```diff
@@ -44,7 +44,7 @@ script:
     - docker exec -it clear-test bash -c "cd /travis ; make dist-clean"
     - docker exec -it clear-test bash -c "cd /travis ; make"
     - docker exec -it clear-test bash -c "cd /travis ; PATH=${PATH}:${GOPATH}:/go/bin make vendor-check"
-    - docker exec -it clear-test bash -c "cd /travis ; make lint"
+    - travis_wait 30 sleep infinity & docker exec -it clear-test bash -c "cd /travis ; make lint"
```

Comment:
So we are keeping lint deadline as 20m and travis_wait as 30m? Just checking if this is intentional.

Reply:
No, they should be consistent.